server/graphql/v2/object/VirtualCard.ts

```diff
@@ -7,6 +7,10 @@ import { Currency } from '../enum/Currency';
 import { Account } from '../interface/Account';
 import { Individual } from '../object/Individual';
 
+const canSeeVirtualCardPrivateInfo = (req, collective, virtualCard) =>
+  (req.remoteUser?.isAdminOfCollectiveOrHost(collective) || req.remoteUser?.id === virtualCard.UserId) &&
```

Comment:
Why do we need this new condition? `req.remoteUser?.id === virtualCard.UserId`. 

Why would the user not be ADMIN of the Collective?

Reply:
This is no longer needed as I've removed the virtual card section from user settings.